### ISSUE TITLE
Downloading from Escapist has recently stopped working

### ISSUE THREAD

**Issue by lbalker** [rg3/youtube-dl#5028]
Seems they added an extra space, so:

```
--- escapist.py~	2015-02-21 12:13:26.000000000 +0100
+++ escapist.py	2015-02-22 18:01:56.906826221 +0100
@@ -42,7 +42,7 @@
         title = raw_title.partition(' : ')[2]
 
         config_url = compat_urllib_parse.unquote(self._html_search_regex(
-            r'<param name="flashvars" value="config=([^"&]+)', webpage, 'config URL'))
+            r'<param name="flashvars" +value="config=([^"&]+)', webpage, 'config URL'))
 
         formats = []
```

[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.escapistmagazine.com/videos/view/zero-punctuation/10038-Grim-Fandango-Does-It-Hold-Up']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.21
[debug] Python version 2.7.3 - Linux-3.2.0-4-amd64-x86_64-with-debian-7.8
[debug] exe versions: avconv 0.8.16-6, avprobe 0.8.16-6, ffmpeg 0.8.16-6, ffprobe 0.8.16-6
[debug] Proxy map: {}
[Escapist] 10038: Downloading webpage
ERROR: Unable to extract config URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/lbr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/home/lbr/bin/youtube-dl/youtube_dl/extractor/common.py", line 269, in extract
    return self._real_extract(url)
  File "/home/lbr/bin/youtube-dl/youtube_dl/extractor/escapist.py", line 45, in _real_extract
    r'<param name="flashvars" value="config=([^"&]+)', webpage, 'config URL'))
  File "/home/lbr/bin/youtube-dl/youtube_dl/extractor/common.py", line 551, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "/home/lbr/bin/youtube-dl/youtube_dl/extractor/common.py", line 541, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract config URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

**Comment by dstftw** [comment]
Thanks for the report. This will be fixed in the next version.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl version 2015.02.24 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
